tests/integrations/test_dsm.py

```diff
@@ -176,6 +176,42 @@ def test_dsm_rabbitmq(self):
         )
 
 
+@features.datastreams_monitoring_support_for_sqs
+@scenarios.integrations
+class Test_DsmSQS:
+    """ Verify DSM stats points for Sqs """
+
+    def setup_dsm_sqs(self):
+        self.r = weblog.get("/dsm?integration=sqs", timeout=60)
```

Comment:
is the integration 'sqs' or 'botocore.sqs' ?  (either is fine, probably a thought for later)

Reply:
IMO it should be sqs since thats the service name, botocore is specific to python